docs/diagnostics/PX1035.md

```diff
@@ -9,25 +9,17 @@ This document describes the PX1035 diagnostic.
 
 ## Diagnostic Description
 A DAC should not have multiple primary, foreign, or unique key declarations which include the same fields. 
-The warning is shown only for DACs that have the `PXCacheName` or `PXPrimaryGraph` attribute.
-The diagnostic does not check abstract DACs and DACs which have only unbound DAC properties (fully-unbound DACs used for custom popups and filters for inquiry forms).
+The diagnostic does not check abstract DACs.
 
-The diagnostic also checks the declaration and implementation of a unique key by the following rules:
- - A single unique key in a DAC declaration must have the name `UK`.
- - Multiple unique keys can have arbitrary names but they must be declared inside a public static class named `UK`.
- - Primary and unique key declarations cannot have the same set of fields.
- - There cannot be a unique key declaration without a primary key declaration. You should turn at least one unique key into a primary key. Acuminator searches for an appropriate candidate for a primary key amoung the DAC key fields by the following rules:
-   - If there is one key field in the DAC, Acuminator suggests to turn it into a primary key named `PK` by renaming and optionally changing its location to make it a first DAC's type member. This is actual for unique keys which are not direct DAC type members because they are declared in the `UK` container class.
-   - If there is no appropriate unique key, Acuminator shows an error indicating the missing primary key.
-
-To fix the issue, you should remove all primary, foreign, or unique key declarations except one and declare unique keys as described in the rules above.
+To fix the issue, you should remove all primary, foreign, or unique key declarations except one.
```

Comment:
Imho, it's better specify you should remove all primary, foreign, or unique key declarations **with same sets of fields**
```suggestion
To fix the issue, you should remove all primary, foreign, or unique key declarations with same sets of fields except one.
```